src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -24,757 +24,757 @@ public static partial class CpuMathUtils
         public static int GetVectorAlignment()
             => Avx.IsSupported ? Vector256Alignment : (Sse.IsSupported ? Vector128Alignment : FloatAlignment);
 
-        public static void MatTimesSrc(bool tran, bool add, AlignedArray mat, AlignedArray src, AlignedArray dst, int crun)
+        public static void MatTimesSource(bool transpose, bool add, AlignedArray mat, AlignedArray source, AlignedArray destination, int crun)
```

Comment:
`mat` here stands for `matrix`

Reply:
Oh, thanks! I'll update that.

Reply:
`crun` is also attempting to indicate how many columns are in each row of the matrix. We could probably have a better name here.... `stride` is the term I'm more generally familiar with.